server/schedule/operator/step.go

```diff
@@ -42,7 +42,10 @@ type OpStep interface {
 
 // TransferLeader is an OpStep that transfers a region's leader.
 type TransferLeader struct {
+	// Compatible with old TiKV's TransferLeader.
 	FromStore, ToStore uint64
+	// Multi-target transfer leader.
+	ToStores []uint64
```

Comment:
Perhaps, we can remove ToStore?

Reply:
I don't think that's a good idea. `ToStores` is only used during evicting leader, whose targets don't need to be filtered during `BuldJointXXX` or `BuildNonJointXXX`. It's better to separate `ToStore` and `ToStores` to avoid misleading.

Reply:
cc @HunDunDM

Reply:
separate 2 fields +1